### PR TITLE
Switch to webpack via wordpress-scripts and simplify build

### PR DESCRIPTION
**Pull request by jom**
### Changes proposed in this Pull Request

* Standardizes `npm` scripts with core Sensei.
* Switches to `wordpress-scripts` (webpack) for building assets.
* Removes gulpfile.js and dependencies.
* Breaking Dev Change: No longer using `build` directory.

### Testing instructions

* Run `npm install` and `npm run build` and test generated `sensei-content-drip.zip` file.

### CODE REVIEW

File: assets/css/admin-lesson.css [hunk 1]
```diff
@@ -1,19 +0,0 @@
-/**
```
Comment (gkaragia) [inline assets/css/admin-lesson.css:1]: This file is probably needed as the UI has an issue:
Screenshot 2020-09-02 at 1 30 47 PM

**Review by gikaragia — COMMENTED** [review]
There is probably some CSS missing.

File: assets/css/admin-lesson.css (hunk 1 above)
Comment (jom) [reply assets/css/admin-lesson.css:1]: @gkaragia It is still there, just moved to `admin-lesson.scss`. Git didn't track it as a rename for some reason. I think this is a bug in the style. When I inspect the interval number field, I can see the style applied from the generated `admin-lesson.scss`.

Comment (jom) [reply assets/css/admin-lesson.css:1]: A-ha! I see what happened. There was already an `admin-lesson.scss` file, however the css file was getting updates that the SCSS file wasn't. I'll bring the changes over to the scss file.

**Review by gikaragia — APPROVED** [review]
Issue is fixed now! I know it is an existing issue, but is it possible to add some space between the button and the input field to make it conistent with the other options?
Screenshot 2020-09-08 at 12 40 44 PM

**Comment by jom** [comment]
@gkaragia We were surrounding `<div>` with `<p>`. I set the style for the `<div>`s and seems to look better. https://github.com/woocommerce/sensei-content-drip/pull/201/commits/7381da45752d90a63dfb9ac46c32d63ed55cf8c8

**Review by gikaragia — APPROVED** [review]
LGTM!

**Pull request merged**